README.md

```diff
@@ -21,17 +21,47 @@ python -m s4.train --dataset mnist-classification --model s4 --epochs 10 --bsz 1
 #### CIFAR-10 Classification
 
 ```bash
+## Adding a Cubic Decay Schedule for last 70% of training
+
+# Default arguments (100 epochs for CIFAR)
+python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 128 --d_model 128 --ssm_n 64 --lr 1e-2 --lr_schedule
+
+# S4 replication from central repository
+python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 64 --d_model 512 --ssm_n 64 --lr 1e-2 --lr_schedule
+
+## After Fixing S4-Custom Optimization & Dropout2D (all implemented inline now... can add flags if desired)
+
+# Default arguments (100 epochs for CIFAR)
+python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 128 --d_model 128 --ssm_n 64 --lr 1e-2
+
+# S4 replication from central repository
+python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 64 --d_model 512 --ssm_n 64 --lr 1e-2
+
+## Before Fixing S4-Custom Optimization...
+
 # Default arguments (100 epochs for CIFAR)
 python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 128 --d_model 128 --ssm_n 64
 
-# S4 replication (close enough to params in original repo's example.py -- no LR schedule though!)
+# S4 replication from central repository
 python -m s4.train --dataset cifar-classification --model s4 --epochs 100 --bsz 64 --d_model 512 --ssm_n 64
 ```
 
-(Default Arguments): Gets "best" 65.51% accuracy @ 46s/epoch on a TitanRTX
+Adding a Schedule:
```

Comment:
Not sure I understand these, is this comparing our implementation to the main one? 

Also is the speed relatively similar?

Reply:
Sorry this is unclear. Running with the same arguments as the main codebase (larger `d_model = 512`) takes forever, I found that running with a smaller model was nice to prototype changes a bit faster, and correlated pretty well. 

I just dropped the different results in the README -- "default args" = smaller model, "S4 replication" = bigger model (as apples-to-apples with original code as possible).

> Also is the speed relatively similar?

Albert/Karan report 7m20s per CIFAR epoch with their model on a T4 GPU, and ~2m30 on an A100. We're seeing 3m11s on a TitanRTX (in between the two GPUs)... Karan validated that speed checks out.